Bottom: 9786d73a751715bbce0ec9e78d6024fa21af790b
Top:    a715d87998a4b5d749a48f4103a42d7535c6bdd4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-30 14:43:27 -0500

DO NOT SUBMIT: Temporary changes for development

---
diff --git a/docker/buildbot-master/buildbot/openembedded.py b/docker/buildbot-master/buildbot/openembedded.py
index 7e3643f..e303054 100755
--- a/docker/buildbot-master/buildbot/openembedded.py
+++ b/docker/buildbot-master/buildbot/openembedded.py
@@ -5,7 +5,7 @@
 import os
 from buildbot.plugins import *
 
-DEFAULT_BBFLAGS = '-k'
+DEFAULT_BBFLAGS = ''
 
 # Workers
 # The 'workers' list defines the set of recognized buildworkers. Each element is
@@ -84,31 +84,31 @@ schedulers = [
         ],
     ),
 
-    schedulers.Nightly(
-        name="ntel-nightly",
-        branch=None,
-        builderNames=[
-            "ntel_orionlxm",
-            "ntel_orionlx_cpx",
-            "ntel_orionlx_plus",
-            "ntel_orion_io",
-            "ntel_qemux86_64",
-            "ntel_all"
-        ],
-        codebases={
-            '': {
-                'repository': DEFAULT_REPO,
-                'branch': 'morty',
-                'revision': '',
-            }
-        },
-        properties={
-            'clobber': True,
-            'cache': True,
-            'bbflags': DEFAULT_BBFLAGS
-        },
-        hour=22
-    ),
+    # schedulers.Nightly(
+    #     name="ntel-nightly",
+    #     branch=None,
+    #     builderNames=[
+    #         "ntel_orionlxm",
+    #         "ntel_orionlx_cpx",
+    #         "ntel_orionlx_plus",
+    #         "ntel_orion_io",
+    #         "ntel_qemux86_64",
+    #         "ntel_all"
+    #     ],
+    #     codebases={
+    #         '': {
+    #             'repository': DEFAULT_REPO,
+    #             'branch': 'morty',
+    #             'revision': '',
+    #         }
+    #     },
+    #     properties={
+    #         'clobber': True,
+    #         'cache': True,
+    #         'bbflags': DEFAULT_BBFLAGS
+    #     },
+    #     hour=22
+    # ),
 
 ]
 
diff --git a/docker/buildbot-master/buildbot/ptxdist.py b/docker/buildbot-master/buildbot/ptxdist.py
index 8502067..7f72b99 100644
--- a/docker/buildbot-master/buildbot/ptxdist.py
+++ b/docker/buildbot-master/buildbot/ptxdist.py
@@ -59,16 +59,16 @@ schedulers = [
         ],
     ),
 
-    schedulers.Nightly(
-        name="ptxdist-nightly",
-        branch=None,
-        builderNames=[
-            "current_armeb_xscale",
-            "current_i686",
-            "current_am335x",
-        ],
-        hour=22
-    ),
+    # schedulers.Nightly(
+    #     name="ptxdist-nightly",
+    #     branch=None,
+    #     builderNames=[
+    #         "current_armeb_xscale",
+    #         "current_i686",
+    #         "current_am335x",
+    #     ],
+    #     hour=22
+    # ),
 
     schedulers.Triggerable(
         name="upgrade_i686",
diff --git a/docker/buildbot-master/buildbot/reconfig.sh b/docker/buildbot-master/buildbot/reconfig.sh
new file mode 100755
index 0000000..b4aecd8
--- /dev/null
+++ b/docker/buildbot-master/buildbot/reconfig.sh
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cd /buildbot && buildbot checkconfig && buildbot reconfig
